ibcgo-1-local.toml

```diff
@@ -0,0 +1,19 @@
+type = "cosmos"
```

Comment:
should we move this into a specific folder? Like hyperspace/docs/cosmos?

Reply:
Will move to `config` folder